examples/Using Interact.ipynb

```diff
@@ -32,7 +32,7 @@
    "metadata": {},
    "source": [
     "<div class=\"alert alert-success\">\n",
-    "As of IPython 3.0, the widgets in this notebook won't show up on http://nbviewer.ipython.org. To view the widgets and interact with them, you will need to download this notebook and run it with an IPython Notebook server.\n",
+    "As of IPython 3.0, the widgets in this notebook won't show up on http://nbviewer.ipython.org. To view the widgets and interact with them, you will need to download this notebook and run it with an IPython or Jupyter Notebook server.\n",
```

Comment:
When I read it, I read it as a lower bound.  I see now that it's an upper bound.